imap_processing/ultra/l1c/ultra_l1c_pset_bins.py

```diff
@@ -123,6 +123,110 @@ def get_spacecraft_histogram(
     return hist, latitude, longitude, n_pix
 
 
+def get_helio_histogram(
+    time: NDArray,
+    vhat: NDArray,
+    energy: NDArray,
+    energy_bin_edges: list[tuple[float, float]],
+    nside: int = 128,
+    nested: bool = False,
+) -> tuple[NDArray, NDArray, NDArray, NDArray]:
+    """
+    Compute a 3D histogram of the particle data using HEALPix binning.
```

Comment:
```suggestion
    Compute a 3D histogram of the particle data in the heliospheric frame using HEALPix binning.
```

That may be too long to fit in the 1 line docstring, but I think it's worth clarifying that it's in the heliospheric frame.